app/src/main/res/values/strings.xml

```diff
@@ -300,4 +300,8 @@
         <item quantity="other">View archive (<xliff:g>%1$d</xliff:g> cards)</item>
     </plurals>
     <string name="welcome">Welcome to Catima</string>
+    <string name="validFromDate">Valid from</string>
+    <string name="anyDate">Any date</string>
+    <string name="chooseValidFromDate">Choose Valid from date</string>
```

Comment:
I'm not sure if I've capitalized "Valid from" here correctly, as English is not my native language. Please have a look. The string for the expiry date is "Choose expiry date"